I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/chapter01.rpy", line 87, in script
    jump day_2
ScriptError: could not find label 'day_2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/chapter01.rpy", line 87, in script
    jump day_2
  File "/Applications/renpy-7.4.11-sdk/renpy/ast.py", line 1757, in execute
    rv = renpy.game.script.lookup(target)
  File "/Applications/renpy-7.4.11-sdk/renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'day_2'.

Darwin-21.3.0-x86_64-i386-64bit
Ren'Py 7.4.11.2266
donor 1.0
Thu Feb  3 20:22:21 2022
